app/components/resend-invitation/component.js

```diff
@@ -0,0 +1,24 @@
+import Ember from 'ember';
+
+export default Ember.Component.extend({
+  tagName: '', // tagless component
+
+  store: Ember.inject.service(),
```

Comment:
I'm going to fix this and will make a separate pr to remove the global initializer